docs/sections/cn_workflows.md

```diff
@@ -188,6 +190,21 @@ cd /path/to
 popper run -f wf.yml
 ```
 
+To specify a workspace directory for a step you can adding `dir` in the
```

Comment:
since this is distinct to the workspace, maybe we can have this as a subsection titled `Changing the working directory`, and explain that this is only changing the working directory where the specified command is being executed from.

And we can also add a note that reminds the reader that if the directory is outside the workspace, then anything that gets written to it won't be persisted.

Reply:
Got it! I'll change it.